### PR TITLE
Update using-microsoft-visualstudio-testtools-unittesting-members-in-…

### PR DESCRIPTION
**Pull request by engyebrahim**
related to: https://github.com/microsoft/testfx/issues/1574

### CODE REVIEW

**Comment by Court72** [comment]
@Mikejo5000 

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/test/using-microsoft-visualstudio-testtools-unittesting-members-in-unit-tests.md [hunk 1]
```diff
@@ -239,7 +239,7 @@ public class MyOtherTestClass
 
 It's possible to control the inheritance behavior: only for current class using `InheritanceBehavior.None` or for all derived classes using `InheritanceBehavior.BeforeEachDerivedClass`.
 
-It's also possible to configure whether the class cleanup should be run at the end of the class or at the end of the assembly (current default).
+It's also possible to configure whether the class cleanup should be run at the end of the class or at the end of the assembly (No longer supported starting from MSTest v4 as EndOfClass is the default and only ClassCleanup behavior).
```
Comment (Mikejo5000) [inline docs/test/using-microsoft-visualstudio-testtools-unittesting-members-in-unit-tests.md:242]: Same as other PR. Can we make "ClassCleanup" a link? I don't think it's mentioned elsewhere in this doc, so might be confusing.

Comment (Mikejo5000) [inline docs/test/using-microsoft-visualstudio-testtools-unittesting-members-in-unit-tests.md:242]:
```suggestion
It's also possible to configure whether the class cleanup should be run at the end of the class or at the end of the assembly (No longer supported starting from MSTest v4 as EndOfClass is the default and only <xref:Microsoft.VisualStudio.TestTools.UnitTesting.ClassCleanupAttribute> behavior).
```

**Comment by Evangelink** [comment]
@Mikejo5000 MSTest v4 is not yet available so maybe you want to keep this PR on-hold for now. We will ping you once v4 is released.

Note that if that's fine for you, I don't mind merging it now.

**Comment by Mikejo5000** [comment]
@Evangelink Is there a planned release date?

**Comment by Evangelink** [comment]
@Mikejo5000 Not at the moment. We are discussing planning so I hope to be able to give you more info soon.

**Comment by engyebrahim** [comment]
Hi @Mikejo5000 we don't know yet the release date, but yeah we are sure it'll be part of v4 so you can merge it. Thanks!!

**Comment by Jak-MS** [comment]
@engyebrahim @Mikejo5000 - without staging, we can't test the xref: link - which looks okay but should be checked after publish. thanks.

**Pull request merged**